### DISCUSSION TITLE
Passthrough USB devices?

### DISCUSSION THREAD

**Discussion by stevefan1999-personal** [coder/coder#8960] [Feature Request]
I want to passthrough my ESP32 so I can run esp-rs on Coder to debug the firmware, but I'm not sure how to do that.

**Comment by hannes09** [comment]
I usually use a USB to Ethernet dongle to access hardware, but Im not sure your ESP32 has TCP/IP support. 
You can try to set a fixed IP address based on the USB dongle MAC adresss and attach the ESP32 to a machine/network that is accessible from your coder workspace.
